doc/_templates/footer.html

```diff
@@ -0,0 +1,41 @@
+<footer>
+  {% if (theme_prev_next_buttons_location == 'bottom' or theme_prev_next_buttons_location == 'both') and (next or prev) %}
+    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
+      {% if next %}
+        <a href="{{ next.link|e }}" class="btn btn-neutral float-right" title="{{ next.title|striptags|e }}" accesskey="n" rel="next">{{ _('Next') }} <span class="fa fa-arrow-circle-right"></span></a>
+      {% endif %}
+      {% if prev %}
+        <a href="{{ prev.link|e }}" class="btn btn-neutral float-left" title="{{ prev.title|striptags|e }}" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> {{ _('Previous') }}</a>
+      {% endif %}
+    </div>
+  {% endif %}
+
+  <hr/>
+
+  <div role="contentinfo">
+    <a href="http://umich.edu">
+    <img src="{{ pathto('_static/umich-block-M.svg', 1) }}" alt="University of Michigan logo" style="width: 100px; float: right;" />
+    </a>
+
+    <p>{{ project }} development is led by the <a href="https://glotzerlab.engin.umich.edu">Glotzer Group</a> at the <a href="https://umich.edu">University of Michigan</a>
```

Comment:
The order will probably be flipped to `Development of {{ project }}` for freud and signac. As a stylistic rule, we avoid starting sentences with the (lowercase) package name.

Reply:
Do we wish to be known as the Glotzer Group or the Glotzer Lab? Our naming is pretty inconsistent between the URL and page titles. I think @kjcoulter gave a guideline about this once but I can't recall it.

Reply:
I would add a period at the end of this sentence.